server/src/main/java/org/opensearch/common/cache/tier/DiskTierRequestStats.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.cache.tier;
+
+/**
+ * A class created by disk tier implementations containing disk-specific stats for a single request.
+ */
+public class DiskTierRequestStats implements TierRequestStats {
+
+    private final long requestGetTimeNanos;
+    private final boolean requestReachedDisk;
```

Comment:
Rename to isDiskCacheHit?

Reply:
We also want to include disk cache misses where we actually read/write the disk though. We use this so we can separate out get() calls where the keystore prevented us from reaching the disk, as we don't want to count those when calculating average disk seek time